src/numeric.rs

```diff
@@ -12,7 +12,8 @@ impl Interval {
     ///  \end{cases}
     /// $$
     ///
-    /// The exact value is returned.
+    /// The exact value is returned.  Note that, as mandated by
+    /// IEEE-1788, when $a$ is ±0., `-0.` is returned.
```

Comment:
How about this:

```
When $a = 0$, `-0.0` is returned.
```

I don't feel the need for mentioning IEEE-1788 here, since all functions are designed to be compliant to the standard.

Reply:
Fine with me.